projects.md

```diff
@@ -0,0 +1,12 @@
+# Chisel Working Group (CWG) Projects
+
+## Current Projects:
+- FIRRTL (https://github.com/freechipsproject/firrtl)
+- Chisel 3 (https://github.com/freechipsproject/chisel3)
+- Chisel Testers (https://github.com/freechipsproject/chisel-testers)
+- Treadle (https://github.com/freechipsproject/treadle)
+- Bootcamp (https://github.com/freechipsproject/chisel-bootcamp)
+- Template (https://github.com/freechipsproject/chisel-template)
```

Comment:
Isn't this one already public domain?

Reply:
It's currently unlicensed but it was included in the relicensing emails so will be relicensed to Apache 2.0.

Reply:
Or rather I mean it should be public domain by principle, not by accident.

Reply:
The template should not be Apache 2.0 but public domain to facilitate adoption.

Reply:
Right, good point. It should be left that way.

Reply:
Public domain is not a thing in many jurisdictions. It should always have a fallback license like Apache-2.0

Reply:
It already has the http://unlicense.org/ applied to it which is essentially a 'legal' version of public domain. Apache 2.0 is NOT public domain.